modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DefaultDLEditFileEntryDisplayContext.java

```diff
@@ -86,6 +90,32 @@ public DLFilePicker getDLFilePicker(String onFilePickCallback) {
 		return null;
 	}
 
+	@Override
+	public String getFriendlyURLBase() {
+		ThemeDisplay themeDisplay =
+			(ThemeDisplay)_httpServletRequest.getAttribute(
+				WebKeys.THEME_DISPLAY);
+
+		StringBundler sb = new StringBundler(4);
+
+		sb.append("/documents");
```

Comment:
@adolfopa 
I did not find any constant to use this.

I do not know if we can move it to DLFileEntryConstants or FriendlyURLResolverConstants or similar.
but we can move it on next tickets

Reply:
Don't worry. It is fine as is.